src/s2wasm.h

```diff
@@ -72,7 +75,7 @@ class S2WasmBuilder {
   };
   std::vector<Relocation> relocations;
 
-  std::set<Name> implementedFunctions;
+  std::map<Name, Function*> implementedFunctions;
```

Comment:
unless i'm missing something, i don't think you need this. you can access `wasm.functionsMap[name]` to get the Function object for a function name.